docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -29,6 +29,54 @@ different flags and/or different memory and cpu requests for different hardware
 
 ## Writing a DaemonSet Spec
 
+###Create a DeamonSet
+
+You can describe a DeamonSet in a YAML file. For example, this  file [deamonset.yaml](deamonset.yaml) describes a DeamonSet that runs the fluentd-elasticsearch Docker image:
```

Comment:
You should explicitly set the file path, i.e. `[deamonset.yaml](/docs/concepts/workloads/controllers/deamonset.yaml)`